src/models/ledger/pay_channel.ts

```diff
@@ -0,0 +1,18 @@
+export interface PayChannel {
+  LedgerEntryType: 'PayChannel'
+  Account: string
+  Destination: string
+  Amount: string
+  Balance: string
+  PublicKey: string
+  SettleDelay: number
+  OwnerNode: string
+  PreviousTxnID: string
+  PreviousTxnLgrSeq: number
+  Flags: number
+  Expiration?: number
+  CancelAfter?: number
+  SourceTag?: number
+  DestinationTag?: number
+  index: string
```

Comment:
Seems like you might be missing `DestinationNode?: string` (Last entry here: https://xrpl.org/paychannel.html#paychannel)